vimms/Controller/dia.py

```diff
@@ -176,3 +171,96 @@ def _process_scan(self, scan):
             self.scan_number += len(precursor_mz_list) + 1
             self.next_processed_scan_id = self.scan_number
         return new_tasks
+
+
+class DiaController(Controller):
```

Comment:
Could you add some comments to our future self to just clarify the difference between this and the other DIA ones?

Reply:
comments have been added in the latest commit